test/system/src/test/resources/logback-test.xml

```diff
@@ -16,7 +16,7 @@ You may obtain a copy of the License at
         </encoder>
     </appender>
 
-    <root level="TRACE">
+    <root level="DEBUG">
```

Comment:
The jenkins job or command line can specify the log level  using `-Dlog.level=DEBUG`

The log level is not passed to the test pods and we should ensure the system properties on the test pod are updated correctly.

https://github.com/pravega/pravega/blob/38b1b001f896c39748cc2cd084a834f6ed28c804/test/system/src/main/java/io/pravega/test/system/framework/services/kubernetes/K8SequentialExecutor.java#L123-L124

https://github.com/pravega/pravega/blob/38b1b001f896c39748cc2cd084a834f6ed28c804/test/system/src/main/java/io/pravega/test/system/framework/DockerBasedTestExecutor.java#L173-L176

Reply:
Addressed the comments.